crates/hyperqueue/src/client/output/cli.rs

```diff
@@ -534,6 +535,66 @@ impl Output for CliOutput {
         );
     }
 
+    fn print_job_cat(&self, job: JobDetail, stdout: bool, stderr: bool) {
```

Comment:
Let's rename this method to `print_job_output` and create an enum, e.g. `OutputStream` that will contain `Stdout` and `Stderr` variants.

Also please make this method return a `Result` and return an error from the JSON printer. It should not support this output, because the data might be binary and it's not clear how to put it into JSON. And if someone uses the JSON output mode, they can process the files themselves.

The quiet mode should have the same implementation as CLI. Either reuse some function that will be called from both Quiet and CLI or just put the implementation as a default impl into the trait.